nativelink-service/src/bytestream_server.rs

```diff
@@ -115,11 +116,11 @@ impl Drop for ActiveStreamGuard<'_> {
             );
             return;
         };
-        let sleep_fn = self.bytestream_server.sleep_fn.clone();
+        let timeout = self.timeout;
         active_uploads_slot.1 = Some(IdleStream {
             stream_state,
-            _timeout_streaam_drop_guard: spawn!("bytestream_idle_stream_timeout", async move {
-                (*sleep_fn)().await;
+            _timeout_stream_drop_guard: spawn!("bytestream_idle_stream_timeout", async move {
+                sleep(timeout).await;
```

Comment:
Why the change? Doesn't feel related to the other bytestream work?

Reply:
I agree